static/scripts/audit-report/audit.ts

```diff
@@ -32,6 +31,12 @@ import {
 } from "./types";
 import { getTxInfo } from "./utils/getTransaction";
 
+if (!process.env.SUPABASE_URL || !process.env.SUPABASE_ANON_KEY) {
```

Comment:
You shouldn't need the process.env if you pass these in with `esbuild` define.  

Refer to my implementation

https://github.com/ubiquity/pay.ubq.fi/pull/167

Reply:
It just substitutes the given value by define in the code, thus substituting `process.env.SUPABASE_URL` by the given value. Process is actually never invoked nor needed, it's more by convention that I called it this way. I will replace it then.